packages/client/src/utils/referenceApi.ts

```diff
@@ -94,6 +106,40 @@ async function loadConfig(): Promise<IApplicationConfig> {
   return response
 }
 
+async function loadCertificatesTemplatesDefinitions(): Promise<
+  ICertificateResponse
+> {
+  const url = `${window.config.CONFIG_API_URL}/getActiveCertificates`
+
+  const res = await fetch(url, {
+    method: 'GET',
+    headers: {
+      Authorization: `Bearer ${getToken()}`
+    }
+  })
+
+  if (res && res.status !== 200) {
+    throw Error(res.statusText)
+  }
+  const response = await res.json()
```

Comment:
Can you add a type for this variable? That way you don't have to use `any` below

Reply:
Yeah. I have declared an interface for this.